CMakeLists.txt

```diff
@@ -108,10 +108,17 @@ if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU" OR "${CMAKE_CXX_COMPILER_ID}" STREQU
     return()
 endif()
 
+# Detect use of the clang-cl.exe frontend, which does not support all of clangs normal options
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" AND "${CMAKE_CXX_SIMULATE_ID}" STREQUAL "MSVC")
+    set(CLANG_CL true)
```

Comment:
Let's reuse the `CLANG_CL` var [here](https://github.com/google/filament/blob/master/filament/CMakeLists.txt#L369)